scraper/src/coordinates_scraper.py

```diff
@@ -0,0 +1,52 @@
+import requests
+import os
+
+def start_processing():
+    data = fetch_data()
+    if (len(data) > 0):
+        transformed_data = transform_data(data)
+
+
+def fetch_data() -> list:
+    data = []
+    url = "https://life-api.coronasafe.network/data/oxygen_v2.json"
+    response = requests.get(url)
+    if ("data" in response.json() and len(response.json()["data"]) > 0):
+        data.extend(response.json()["data"])
+    return data
+
+def transform_data(data):
```

Comment:
IMHO, we should write lat long back to db and have a filter to NOT update lat long if values are already updated.
Address changes need to be handled.

Reply:
We sure want to update the DB with this data so that this is an operation thats run fewer times and not every x mins like rest of the scrapper functions are running. This is first cut and I'll hold off on merging it till we bring it to db integration.